packages/web-components/src/components/va-official-gov-banner/va-official-gov-banner.tsx

```diff
@@ -0,0 +1,209 @@
+  }
+
+  componentDidLoad(){
+    // Initial loading of formatted text.
+    this.govSiteExplanationText();
+    this.govSiteLockText();
+  }
+
+  private handleClick = () => {
+    const content = this.el.shadowRoot?.querySelector('.content') as HTMLElement;
+    const button = this.el.shadowRoot?.querySelector('button') as HTMLElement;
+    const header = this.el.shadowRoot?.querySelector('header') as HTMLElement;
+
+    button.setAttribute(
+      'aria-expanded', 
+      button.getAttribute('aria-expanded') === 'true' 
+        ? 'false' 
+        : 'true'
+    );
+
+    header.classList.toggle('expanded');
+    // Toggle the hidden attribute on the content.
+    content.hidden ? content.removeAttribute('hidden') : content.setAttribute('hidden', 'true');
+
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-official-gov-banner',
+        action: 'click',
+        details: { 
+          event: 'int-official-gov-banner-expand',
+        },
+      }
+      this.componentLibraryAnalytics.emit(detail);
+    }
+  };
+
+  /**
+   * This  will add <strong> tags around the 
+   * matching TLD value (.gov or .mil) in the text that is provided
+   * and the innerHTML of the element will be updated.
+   */
+  private govSiteExplanationText = () => {
+    const el = this.el.shadowRoot?.querySelector('.gov-site-explanation-text') as HTMLElement;
+    if (el) {
+      const text = i18next.t('gov-site-explanation', { tld: this.tld });
+      el.innerHTML = text.replaceAll(`.${this.tld}`, `<strong>.${this.tld}</strong>`);
+    }
+  }
+
+  /**
+   * This will add <strong> tags around various words in the text that is provided
+   * as well as add the SVG lock image.
+   */
+  private govSiteLockText = () => {
+    const el = this.el.shadowRoot?.querySelector('.gov-site-lock-text') as HTMLElement;
+
+    // eslint-disable-next-line i18next/no-literal-string
+    const lockSvg = `(&nbsp;<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" role="img" aria-labelledby="banner-lock-description" focusable="false">
+    <title id="banner-lock-title">Lock</title>
+    <desc id="banner-lock-description">Locked padlock icon</desc>
+    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
+    </svg></span>&nbsp;)`
+  
+    let html = i18next.t('gov-site-lock', { image: 'SVG', tld: this.tld });
+
+    html = html.replace('SVG', lockSvg);
+    html = html.replace('lock', `<strong>lock</strong>`);
+    html = html.replace('candado', `<strong>candado</strong>`);
+    html = html.replace('https://', `<strong>https://</strong>`);
+
+    if (el) {
+      el.innerHTML = html;
+    }
+  }
+
+  render() {
+    const { tld } = this;
+    if (tld === 'gov' || tld === 'mil') {
```

Comment:
I experimented with getting the top-level domain value programmatically within the component itself but found that it would be very difficult to test or display the component on sites that weren't actually `.gov` or `.mil` sites (like Chromatic and Storybook). If we're not happy with relying on the developer to set the correct value here manually, then maybe we can discuss alternatives. The default prop value is `.gov`.